packages/cfl/tests/test_finding.py

```
@@ -0,0 +1,917 @@
+    ).fetchone()["cnt"]
+    assert count == 0
+
+
+# ---------------------------------------------------------------------------
+# Vocabulary constants are exported
+# ---------------------------------------------------------------------------
+
+
+def test_known_severities_exported():
+    """KNOWN_SEVERITIES matches challenge's severity taxonomy."""
+    assert KNOWN_SEVERITIES == frozenset({"CRITICAL", "HIGH", "MEDIUM", "TENSION"})
```

Ruff (0.16.1)

[warning] Yoda condition detected

(SIM300)